spec/mainadditions.html

```diff
@@ -45,19 +48,190 @@ <h1>Mathematical Operations</h1>
+        <h1>
+          UTC (
+            _t_: a Number,
+          ): a time value
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>
+            It converts _t_ from local time to a UTC time value.
+            <ins>The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.</ins>
+          </dd>
+        </dl>
+        <emu-alg>
+          1. <ins>Let _localTimeZone_ be ! CreateTemporalTimeZone(DefaultTimeZone()).</ins>
+          1. <ins>If _localTimeZone_.[[OffsetNanoseconds]] is not *undefined*, then</ins>
+            1. <ins>Let _offsetNs_ be -_localTimeZone_.[[OffsetNanoseconds]].</ins>
+          1. <ins>Else,</ins>
+            1. <ins>Let _year_ be ℝ(YearFromTime(_t_)).</ins>
+            1. <ins>Let _month_ be ℝ(MonthFromTime(_t_)) + 1.</ins>
+            1. <ins>Let _day_ be ℝ(DateFromTime(_t_)).</ins>
+            1. <ins>Let _hour_ be ℝ(HourFromTime(_t_)).</ins>
+            1. <ins>Let _minute_ be ℝ(MinFromTime(_t_)).</ins>
+            1. <ins>Let _second_ be ℝ(SecFromTime(_t_)).</ins>
+            1. <ins>Let _millisecond_ be ℝ(msFromTime(_t_)).</ins>
+            1. <ins>Let _possibleInstants_ be GetIANATimeZoneEpochValue(_localTimeZone_.[[Identifier]], _year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, 0, 0).</ins>
+            1. <ins>If _possibleInstants_ is not empty, then</ins>
+              1. <ins>Let _disambiguatedInstant_ be _possibleInstants_[0].</ins>
+            1. <ins>Else,</ins>
+              1. <ins>Let _epochNs_ be GetEpochFromISOParts(_year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, 0, 0).</ins>
+              1. <ins>Let _dayBefore_ be _epochNs_ - *8.64 &times; 10<sup>13</sup>*<sub>ℤ</sub>.</ins>
+              1. <ins>Let _dayAfter_ be _epochNs_ + *8.64 &times; 10<sup>13</sup>*<sub>ℤ</sub>.</ins>
+              1. <ins>Let _offsetBefore_ be GetIANATimeZoneOffsetNanoseconds(_dayBefore_, _localTimeZone_.[[Identifier]]).</ins>
+              1. <ins>Let _offsetAfter_ be GetIANATimeZoneOffsetNanoseconds(_dayAfter_, _localTimeZone_.[[Identifier]]).</ins>
+              1. <ins>Let _nanoseconds_ be _offsetAfter_ - _offsetBefore_.</ins>
+              1. <ins>Let _balanceResult_ be ! BalanceISODateTime(_year_, _month_, _day_, _hour_, _minute_, _second_, _millisecond_, 0, _nanoseconds_).</ins>
+              1. <ins>Set _possibleInstants_ to GetIANATimeZoneEpochValue(_localTimeZone_.[[Identifier]], _balanceResult_.[[Year]], _balanceResult_.[[Month]], _balanceResult_.[[Day]], _balanceResult_.[[Hour]], _balanceResult_.[[Minute]], _balanceResult_.[[Second]], _balanceResult_.[[Millisecond]], _balanceResult_.[[Microsecond]], _balanceResult_.[[Nanosecond]]).</ins>
+              1. <ins>Assert: _possibleInstants_ is not empty.</ins>
+              1. <ins>Let _disambiguatedInstant_ be the last element of _possibleInstants_.</ins>
```

Comment:
@gibson042 I've pushed an attempt at this prose without using BalanceISODateTime which I wanted to avoid pulling into ECMA-262 if we are able to stage this for inclusion in the spec prior to Temporal. Would you mind taking a look?